exercisePPJ_05_VI/ExercisePPJ_05_VI.java

```diff
@@ -0,0 +1,34 @@
+/*
+Sprawdz jaki będzie rezultat poniższego programu: 
+int x = 4 ;
+long y = x ∗ 4 − x++;
+if ( y<12) System . out . println ( "za malo " ) ; 
+else System . out . println ( "w sam raz" ) ;
+Odpowiedź uzasadnij.
+*/
+
+public class ExercisePPJ_05_VI {
+	
+	public static void main(String[] args) {
+	
+	int x1 = 4 ;
+	long y1 = x1 * 4 - x1++;
```

Comment:
does it make you wonder why `x1` and `y1` are here?
I'm asking because it seem redundant to me. (I know it's part of the task, just saying)

Reply:
@MatDls 
y1 and y2 have a different value, in this situation code must contain 2 variables. x1 and x2 are redundant - yes. I will change this.